packages/server/src/automations/tests/utilities/index.ts

```diff
@@ -87,7 +131,8 @@ export async function captureAllAutomationResults(
   }
   const messageListener = async (message: Job<AutomationData>) => {
     // Don't count cron messages, as they don't get triggered automatically.
-    if (message.opts?.repeat != null) {
+    const isManualTrigger = (message as any).manualTrigger === true
```

Comment:
Could we capture this `manualTrigger` to a type, of is `Job` too attach to bull?